test/bddtests/fixtures/config/mutual_tls/ca/client_sdk_go-ca_root.pem

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
should also use generated fixtures. these should not be checked in.

Reply:
this is generated by open ssl

Reply:
take a look at the "tls.example.com" organization in the fabric-sdk-go.

Reply:
done